guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -27,52 +27,64 @@ endif::[]
+. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
 endif::[]
-. If you want to use `wget` to register your host to {Project}, select the *Use wget* option.
-By default, {Project} generates a `curl` command.
-. If your host does not trust the SSL certificate of your {ProjectServer}, select the *Insecure* option.
-During the first call, your host downloads the CA file from {Project}.
-Your host will use this CA file to connect to {ProjectServer} with all future calls.
+. Optional: Select the *Insecure* option, if you want to make the first call insecure.
+During this first call, hosts download the CA file from {Project}.
+Hosts will use this CA file to connect to {Project} with all future calls making them secure.
 +
 {Team} recommends that you avoid insecure calls.
 +
-If an attacker, located in the network between {Project} and your host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from your host and the JSON Web Tokens (JWT).
-Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access your host using the SSH key.
+If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
+Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
 +
-Instead, you can manually copy and install the CA file on each host before registration.
+Instead, you can manually copy and install the CA file on each host before registering the host.
 +
 To do this, find where {Project} stores the CA file by navigating to *Administer* > *Settings* > *Authentication* and locating the value of the *SSL CA file* setting.
 +
-Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on your host and enter the following commands:
+Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on hosts and enter the following commands:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # update-ca-trust enable
 # update-ca-trust
 ----
++
+Then register the hosts with a secure `curl` command, such as:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# curl -sS https://{foreman-example-com}/register ...
+----
++
+The following is an example of the `curl` command with the `--insecure` option:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# curl -sS --insecure https://{foreman-example-com}/register ...
+----
 . Select the *Advanced* tab.
-. Optional: From the *Setup REX* list, select whether you want to deploy {Project} SSH keys to your host or not.
+. Optional: From the *Setup REX* list, select whether you want to deploy {Project} SSH keys to hosts or not.
```

Comment:
You can remove "or not". Using "whether" implies that there are 2 options. "... select whether to deploy {Project} SSH keys to the hosts".